### ISSUE TITLE
Disable data-sveltekit-preload-data for language selection

### ISSUE THREAD

**Issue by jnsprnw** [ivanhofer/typesafe-i18n-demo-sveltekit-jsdoc#4]
It took me some time to figure that out, so it might help others as well:

Newer versions of Sveltekit (than used in this repo at the moment) [enable link preloading by default](https://kit.svelte.dev/docs/link-options#data-sveltekit-preload-data) in the `app.html` like this:
```html
<body data-sveltekit-preload-data="hover">
```
That’s good for most cases, but it messes around with the language selection from this library.
Instead of completely disabling it, you can also disable it for each link with `data-sveltekit-preload-data="off"`.

`LocaleSwitcher.svelte` can use it like this:
```html
<ul>
  {#each locales as l}
    <li>
      <a data-sveltekit-preload-data="off" type="button" class:active={l === $locale} href={replaceLocaleInUrl($page.url, l)}>
        {l}
      </a>
    </li>
  {/each}
</ul>
```

I hope that helps!

**Comment by ivanhofer** [comment]
Did you check the latest version of the example?

**Comment by jnsprnw** [comment]
Yes, and it works with this setup. But if you create a new Sveltekit website, it will automatically have `<body data-sveltekit-preload-data="hover">` included.

**Comment by ivanhofer** [comment]
But why would you not want to preload the page in the new language when hovering over it?
